rgf/rgf.py

```diff
@@ -378,17 +369,21 @@ def predict_proba(self, X):
         #Find latest model location
         model_glob = loc_temp + os.sep + self.file_prefix + "*"
         if not glob(model_glob):
-            raise Exception('Model learning result is not found @{0}. This is rgf_python error.'.format(loc_temp))
+            raise Exception('Model learning result is not found in @{0}. This is rgf_python error.'.format(loc_temp))
```

Comment:
Model learning result is not found @{0}
or 
Model learning result is not found in {0}

 is preferable. How do you think?